examples/spire/k8s-workload-registrar-service.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: k8s-workload-registrar
+  namespace: spire
+spec:
+  ports:
+    - name: webhook
+      protocol: TCP
+      port: 443
+      targetPort: 9443
+  selector:
+    app: spire-server
```

Comment:
Why do we need a copy of the service for spire-server?

Reply:
ok, done